nzbhydra2: 4.7.6 -> 7.2.3, add maintainer

matteo-pacini opened the pull request:
## Description of changes

- Upgraded nzbhydra2 to the latest available version
- Now requires `Java 17`
- Added myself as maintainer to keep this up-to-date 

- Changelog (huge): https://github.com/theotherp/nzbhydra2/compare/v4.7.6...v7.2.3

![image](https://github.com/NixOS/nixpkgs/assets/3139724/9746a32d-8d94-484c-b2ed-8a66b97e112e)

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

Commit:
nzbhydra2: 4.7.6 -> 7.2.3

Commit:
nzbhydra2: add maintainer

JamieMagee commented:
Result of `nixpkgs-review pr 325062` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>nzbhydra2</li>
  </ul>
</details>

JamieMagee commented:
@matteo-pacini thanks for updating this. I have switched from nzbhydra to prowlarr, so haven't looked at this package in a while.

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-already-reviewed/2617/1802

aanderse commented:
@JamieMagee skins good, thanks for all your work! should we remove you as a maintainer in this PR?
@matteo-pacini thank you for stepping up as maintainer! let's see what @JamieMagee says and then merge

ping me back in a couple days if no updates please

JamieMagee commented:
Yes, please remove me as a maintainer

Aleksanaa committed:
nzbhydra2: remove jamiemagee from maintainers

As requested by @JamieMagee in https://github.com/NixOS/nixpkgs/pull/325062#issuecomment-2212469373

matteo-pacini committed:
nzbhydra2: changed test maintainers

Removed jamiemagee from maintainers.
Added myself as maintainer.

matteo-pacini commented:
@Aleksanaa there was a typo in the package name in your commit, I fixed it

I've also assigned myself to the tests for this package, if that's okay @JamieMagee

aanderse reviewed (approved):
thanks everyone

good to merge now?

JamieMagee commented:
@matteo-pacini sounds good! Thanks for taking over 😁

matteo-pacini commented:
@JamieMagee Happy to help! 💯
